plugins/inventory/cobbler.py

```diff
@@ -0,0 +1,250 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2020 Orion Poplawski <orion@nwra.com>
+# Copyright (c) 2020 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: cobbler
+    plugin_type: inventory
+    short_description: cobbler inventory source
+    description:
+        - Get inventory hosts from the cobbler service.
+        - "Uses a configuration file as an inventory source, it must end in ``.cobbler.yml`` or ``.cobbler.yaml`` and has a ``plugin: cobbler`` entry."
+    extends_documentation_fragment:
+        - inventory_cache
+    options:
+      plugin:
+        description: the name of this plugin, it should always be set to 'cobbler' for this plugin to recognize it as it's own.
+        required: True
+        choices: ['cobbler']
+      url:
+        description: url to cobbler
+        default: 'http://cobbler/cobbler_api'
+        env:
+            - name: COBBLER_SERVER
+      user:
+        description: cobbler authentication user
+        required: False
+        env:
+            - name: COBBLER_USER
+      password:
+        description: cobbler authentication password
+        required: False
+        env:
+            - name: COBBLER_PASSWORD
+      exclude_profiles:
+        description: profiles to exclude from inventory
+        type: list
+        default: []
```

Comment:
```suggestion
        description: Profiles to exclude from inventory.
        type: list
        elements: str
        default: []
```
and in other places
are `elements: ` relevant here? @felixfontein

Reply:
I think so, added.